v0100/srclib/dpstub.asm

```diff
@@ -884,3 +891,4 @@ regs_len equ $ - regs
 fbuf        resb FBUFSZ ; shared with 32-bit code
 
 alignb 16
+end_of_memory:
```

Comment:
This is not the end. The stub is compiled as a small memory model .EXE and an entire 64KB segment is allocated for all of its data and stack (the code segment can be smaller than 64KB). So, between this end_of_memory (offset C2D0H) and the end (offset FFFEH) there's the stack. You can't just tell DOS that it's OK to overwrite the stack.

Reply:
Thanks, now I move the stack
pointer too.